zadani/soft/index.html

```diff
@@ -1,72 +1,86 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>Cestování po Islandu</title>
-        <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400i|Raleway:400,700" rel="stylesheet">
-        <link rel="stylesheet" href="style.css">
-    </head>
-    <body>
 
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Cestování po Islandu</title>
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:400,400i|Raleway:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
 
-        <!-- hlavička -->
+<body>
 
-        <h1 class="titulek">Island</h1>
 
-        <p class="perex">Když se řekne Island, spoustě lidem se jako první vybaví sníh a led. Sama jsem tam strávila téměř rok života a zem pokrytou sněhem a ledem jsem zažila jen jediný den. Jednou věcí si ale jistá jsem: že návštěva ostrova pod polárním kruhem je pro každého tak silným zážitkem, že se sem bude chtít vrátit.</p>
+    <!-- hlavička -->
+    <div class=cela_stranka>
 
-        <p class="autor">Jana Zcela Vymyšlená</p>
+        <div class="hlavicka">
+            <h1 class="titulek">Island</h1>
 
+            <p class="perex">Když se řekne Island, spoustě lidem se jako první vybaví sníh a led. Sama jsem tam strávila téměř rok života a zem pokrytou sněhem a ledem jsem zažila jen jediný den. Jednou věcí si ale jistá jsem: že návštěva ostrova pod polárním kruhem je pro každého tak silným zážitkem, že se sem bude chtít vrátit.</p>
 
+            <p class="autor">Jana Zcela Vymyšlená</p>
 
+        </div>
 
-        <!-- menu -->
 
-        <ul class="menu">
-            <li><a href="#">Úvod</a></li>
-            <li><a href="#">Proč jet na Island</a></li>
-            <li><a href="#">Co musíte vidět</a></li>
-            <li><a href="#">Fotogalerie</a></li>
+</body>
+<!-- menu -->
+
+<div class="obsah">
+
+    <div class="menu">
+        <ul class="menu_seznam">
+            <li><a href="#Úvod">Úvod</a></li>
+            <li><a href="#proc_jet_na_island">Proč jet na Island</a></li>
+            <li><a href="#co_musite_videt">Co musíte vidět</a></li>
+            <li><a href="#fotogalerie">Fotogalerie</a></li>
         </ul>
+    </div>
 
 
+    <!-- sekce Úvod -->
 
-        <!-- sekce Úvod -->
+    <h1 id="Úvod"></h1>
```

Comment:
To bych tu nepřidávala a pokud přece jo, tak jako `h2`. Existuje taková poučka, že by měl být jen 1` h1` na stránce a to už je v našem případě `Island` na řádku 19. A tenhle nadpis mi připadá na stejné úrovni jako ostatní podnadpisy 🙂